sdk/authorization/Azure.ResourceManager.Authorization/CHANGELOG.md

```diff
@@ -1,15 +1,15 @@
 # Release History
 
-## 1.1.0-beta.2 (Unreleased)
+## 1.1.1 (2023-11-21)
```

Comment:
Not sure how we increase the version here, can it be `1.1.0` like we do for `Azure.ResourceManager.Cdn` below?

Reply:
fixed